Spanner/src/Database.php

```diff
@@ -440,6 +441,10 @@ public function exists(array $options = [])
      *     Configuration Options
      *
      *     @type string[] $statements Additional DDL statements.
+     *     @type RetrySettings|array $retrySettings
+     *           Retry settings to use for this call. Can be a {@see RetrySettings} object, or an
+     *           associative array of retry settings parameters. See the documentation on
+     *           {@see RetrySettings} for example usage.
```

Comment:
This needs to be added to each and every API which supports RetrySettings. Note: `executeStreamingSql` and `streamingRead` does not support `RetrySettings`.